__tests__/callbacks.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+jest.mock('fs');
+
+const callbacks = require('../lib/callbacks.js');
```

Comment:
Another naming convention here: generally, try to have the name of this match what action the function does. So perhaps `changeFile` or `callbackChangeFile`